tests/tst_load.cpp

```diff
@@ -306,6 +309,33 @@ void tst_load::testByDate()
     QVERIFY(mStorage->save(ExtendedStorage::PurgeDeleted));
 }
 
+void tst_load::testByAttendee()
+{
+    KCalendarCore::Event::Ptr event(new KCalendarCore::Event);
+    event->setDtStart(QDateTime(QDate(2023,2,2), QTime(15, 35), Qt::UTC));
+    event->addAttendee(KCalendarCore::Attendee(QString::fromLatin1("Alice"),
+                                               QString::fromLatin1("alice@example.org")));
+    event->recurrence()->setDaily(1);
+    QVERIFY(mStorage->calendar()->addEvent(event));
+    KCalendarCore::Incidence::Ptr exception = KCalendarCore::Calendar::createException(event, event->dtStart().addDays(2));
+    exception->clearAttendees();
+    QVERIFY(mStorage->calendar()->addIncidence(exception));
+    QVERIFY(mStorage->save());
+
+    ExtendedCalendar::Ptr calendar(new ExtendedCalendar(QTimeZone::utc()));
+    ExtendedStorage::Ptr storage = ExtendedCalendar::defaultStorage(calendar);
+    QVERIFY(storage->open());
+
+    QVERIFY(calendar->events().isEmpty());
+
+    QVERIFY(storage->loadAttendeeIncidences());
```

Comment:
Right, we had even such a thing. Not sure are these funkier load method of any use for anything. This I don't is needed anywhere. And loadGeoIncidences(), how many events even have coordinate information. 

No objections on adding tests, but could also consider just killing these.

Reply:
I added the test to be sure that at some point in the future, if the function is used, nobody remove the exception loading. If we kill the function anyway in the future, all the best !